### PR TITLE
Add R code and define variables

### PR DESCRIPTION
**Pull request by Hassy2u**
Study definition - Added latest GP consultation date from 120 days prior to study start date to the end of the study period.
Also several shortened variable names.

Additional R script - Added R code to work with input.csv file to calculate variables, including;

1. date of death (maximum date from the 3 causes of death variables/columns),
2. number of days between date of death and latest GP consultation date,
3. and binary flags to show whether the death was within 30 days, 60 days, 90 days, or 120 days of the latest GP consultation date.

### CODE REVIEW

File: analysis/Rcode.R [hunk 1]
```diff
@@ -0,0 +1,118 @@
+library(tidyverse)
+
+#Add data source - input.csv
+file_source <-output/input.csv
+
+data<-read.csv(file = file_source)
```
Comment (milanwiedemann) [inline analysis/Rcode.R]: use relative file path here

File: analysis/Rcode.R [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+xtab60$Days_from_GP <- 60
+xtab90$Days_from_GP <- 90
+xtab120$Days_from_GP <- 120
+
+xtab30<-xtab30[c(5,1, 2:4)]
+xtab60<-xtab60[c(5,1, 2:4)]
+xtab90<-xtab90[c(5,1, 2:4)]
+xtab120<-xtab120[c(5,1, 2:4)]
+
+FinalTable<-rbind(xtab30,xtab60,xtab90,xtab120)
```
Comment (milanwiedemann) [inline analysis/Rcode.R]: needs to be written as a csv file into outputs folder

**Review by milanwiedemann — CHANGES_REQUESTED** [review]

**Comment by milanwiedemann** [comment]
This looks good! I've made a couple suggestions in the pull request #11 
- use relative file paths, check the here R package
- add the R script to the project yaml

I'd also suggest that you give the R script a more meaningful name and change it in the project.yaml

**Review by Hassy2u — COMMENTED** [review]
accepted updated version

**Pull request merged**